src/ucs/vfs/fuse/ucx-fuse.pc.in

```diff
@@ -1,5 +1,11 @@
+prefix = @prefix@
+exec_prefix = @exec_prefix@
+libdir = @libdir@/ucx
+
 Name: @PACKAGE@-fuse
 Description: Unified Communication X Library FUSE module
-Version: @MAJOR_VERSION@.@MINOR_VERSION@
+Version: @VERSION@
 Libs:
-Libs.private:
+Libs.private: -L${libdir} -Wl,--whole-archive,-lucs_fuse,--no-whole-archive 
+Requires: fuse3
+Requires.private: fuse3
```

Comment:
is it needed even though it is identical to `requires`?

Reply:
as I can see yes. in any case pkg-config removes duplicates